src/components/Signup/Signup.js

```diff
@@ -33,6 +35,12 @@ const minLength = {
   errorMessageKey: 'validation_messages.error_min_length_2',
 };
 
+/*Extract the page parameter from url*/
```

Comment:
Tal vez puedas usar JSDoc.

Reply:
ahi no te segui..

Reply:
Ya que ponés un comentario, estaríua bueno que ese comentario siga un formato standard. De esa manera, le va a servir a VS Code para darte info en el intellisense, para generar documentatción, etc.

[JSDoc](https://devdocs.io/jsdoc/) es muy grande y complejo, y aunque creo que no está de más ir aprendiendo, en este caso me parece que el cambio sería muy simple:

```suggestion
/** Extract the page parameter from url */
```

Reply:
Ahh bien, vi que los comentarios estaban asi pero no entendia porque. Gracias por la info!